javascripts/src/lib/grecaptcha.ts

```diff
@@ -1,9 +1,11 @@
 // License: LGPL-3.0-or-later
 
 interface GRecaptcha {
-    execute(widgetIdOrSiteId: string, details?: { action: string }): GRecaptcha.Thenable
+    enterprise: {
+        execute(widgetIdOrSiteId: string, details?: { action: string }): GRecaptcha.Thenable
 
-    ready(callback:()=> void): void
+        ready(callback:()=> void): void
+    }
```

Comment:
Interface of which recaptcha function we need to use has changed here, it used to be `recaptcha.execute()` and `recaptcha.ready()`. On Enterprise, it's `recaptcha.enterprise.execute()` and `recaptcha.enterprise.ready()`.